core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -181,8 +196,112 @@ impl SchedulerController {
         }
     }
 
+    /// Forward packets to the next leader.
+    fn forward_packets(&mut self, hold: bool) {
```

Comment:
is this copypasta from the old scheduler?

Reply:
No this is all new. old scheduler didn't have same container structure, or the same checks we do here (actually checking if tx can pay fees), or timeout.